user.name.exists=Il existe déjà un utilisateur portant cet identifiant.
error=Une erreur interne est survenue. Merci de contacter votre support.
